prow/jobs/generated/knative-sandbox/async-component-main.gen.yaml

```diff
@@ -43,11 +43,14 @@ periodics:
     volumes:
     - name: test-account
       secret:
-        secretName: test-account
+        items:
+        - key: service-account-key.json
+          path: service-account.json
```

Comment:
Are you planning to move these jobs to use workload identity? If yes you may need some changes to Istio `prowgen` since the preset does not support configuring serviceAccoutName now.

Reply:
I noticed that as well.